build.gradle

```diff
@@ -33,6 +35,17 @@ googleJavaFormat {
   exclude 'build/**'
 }
 
+if (project.hasProperty('ossrhUsername') && project.hasProperty('ossrhPassword') &&
```

Comment:
I think we can just remove the if check here. If the user runs the task there would be an error message saying the required fields is not found.